script.js

```diff
@@ -4,12 +4,12 @@ class AudioController {
         this.flipSound = new Audio('Assets/Audio/flip.wav');
         this.matchSound = new Audio('Assets/Audio/match.wav');
         this.victorySound = new Audio('Assets/Audio/victory.wav');
-        this.gameoverSound = new Audio('Assets/Audio/gameover.wav');
+        this.gameOverSound = new Audio('Assets/Audio/gameOver.wav');
         this.bgMusic.volume = 0.5;
+        this.bgMusic.loop = true;

```

Comment:
I moved this to the constructor since it didn't makes sense to have in the `startMusic` function since it is for initializing the audio.